lte/gateway/python/magma/pipelined/app/BUILD.bazel

```diff
@@ -0,0 +1,232 @@
+    ],
+)
+
+py_library(
+    name = "conntrack",
+    srcs = ["conntrack.py"],
+    deps = [
+        ":base",
+        "//lte/gateway/python/magma/pipelined/openflow:flows",
+    ],
+)
+
+py_library(
+    name = "gy",
+    srcs = ["gy.py"],
+    deps = [
+        ":base",
+        ":enforcement_stats",
+        ":inout",
+        "//lte/gateway/python/magma/pipelined:redirect",
+        "//lte/gateway/python/magma/pipelined/qos:qos_meter_impl",
+    ],
+)
+
+py_library(
+    name = "ipv6_solicitation",
+    srcs = ["ipv6_solicitation.py"],
+    deps = [
+        "//lte/gateway/python/magma/pipelined:ipv6_prefix_store",
+        "//lte/gateway/python/magma/pipelined/app:base",
+        "//lte/gateway/python/magma/pipelined/openflow:flows",
+        "//orc8r/gateway/python/magma/common:misc_utils",
+    ],
+)
+
+py_library(
+    name = "startup_flows",
+    srcs = ["startup_flows.py"],
+)
+
+py_library(
+    name = "xwf_passthru",
+    srcs = ["xwf_passthru.py"],
+)
+
+py_library(
+    name = "ng_services",
+    srcs = ["ng_services.py"],
+    deps = ["//lte/gateway/python/magma/pipelined/ng_manager:node_state_manager"],
+)
+
+py_library(
+    name = "enforcement",
+    srcs = ["enforcement.py"],
+    deps = [
+        ":base",
+        ":enforcement_stats",
+        "//lte/gateway/python/magma/pipelined:redirect",
+        "//lte/gateway/python/magma/pipelined/qos:qos_meter_impl",
+    ],
+)
+
+py_library(
+    name = "enforcement_stats",
+    srcs = [
+        "enforcement_stats.py",
+        "policy_mixin.py",
+    ],
+    deps = [
+        "//lte/gateway/python/magma/pipelined:utils",
+        "//lte/gateway/python/magma/pipelined/app:dpi",
+        "//lte/gateway/python/magma/pipelined/app:restart_mixin",
+        "//lte/gateway/python/magma/pipelined/ng_manager:session_state_manager",
+        "//lte/gateway/python/magma/pipelined/qos:types",
+        "//orc8r/gateway/python/magma/common:rpc_utils",
+    ],
+)
+
+py_library(
+    name = "check_quota",
+    srcs = ["check_quota.py"],
+    deps = [":ue_mac"],
+)
+
+py_library(
+    name = "ue_mac",
+    srcs = ["ue_mac.py"],
+    deps = [
+        ":inout",
+        ":ipfix",
+        "//lte/gateway/python/magma/pipelined:directoryd_client",
+    ],
+)
+
+py_library(
+    name = "ipfix",
+    srcs = ["ipfix.py"],
+    deps = [":dpi"],
+)
+
+py_library(
+    name = "dpi",
+    srcs = ["dpi.py"],
+    deps = [
+        ":base",
+        "//lte/gateway/python/magma/pipelined/openflow:flows",
+    ],
+)
+
+py_library(
+    name = "inout",
+    srcs = [
+        "inout.py",
+        "li_mirror.py",
+    ],
+    deps = [
+        ":base",
+        ":restart_mixin",
+        "//lte/gateway/python/magma/pipelined:imsi",
+        "//lte/gateway/python/magma/pipelined:mobilityd_client",
+        "//lte/gateway/python/magma/pipelined:utils",
+        "//lte/gateway/python/magma/pipelined/openflow:flows",
+        "//orc8r/gateway/python/magma/configuration:mconfig_managers",
```

Comment:
Is the `mconfig_managers` really needed here?

Reply:
Same as above, it is not needed for the bazel build now, but it is imported directly from `li_mirror.py`.